tools/cmake_call.sh

```diff
@@ -13,12 +13,19 @@ cd src
 sh ./scripts/nlopt_download.sh ${RSCRIPT_BIN}
 dot() { file=$1; shift; . "$file"; }
 dot ./scripts/r_config.sh ""
+
+# Only R > 4 will have the variables
+$RSCRIPT_BIN --vanilla -e "getRversion() > 4" | grep TRUE > /dev/null
```

Comment:
Can you make it `'getRversion() >= "4.0.0"'` which is more idiomatic and clearer?  (That is outer single quote ' protecting inner double quotes.)  A more elegant way would use the return code as `grep` is a little dirty.  But it does a proper version comparison and is better than grep'ing `Rscript --version`.